frontend/src/stores/editionStore.ts

```diff
@@ -14,13 +14,13 @@ export const useEditionStore = defineStore("editionStore", () => {
   async function loadEdition(): Promise<
     ServiceResponse<LegalPeriodicalEdition>
   > {
-    const uuid = route.params.uuid
+    const uuid = route.params.editionId
     const response = await LegalPeriodicalEditionService.get(
       uuid.toString() as UUID,
     )
 
     if (response.data) {
-      edition.value = response.data
+      edition.value = response.data as LegalPeriodicalEdition
```

Comment:
Unfortunately, this `as` won't work, as it does not how to parse the java object into a typescript object. We need to call the constructor. I changed it in the [commit](https://github.com/digitalservicebund/ris-backend-service/commit/8ba4d1e0f241e89dac90bc6e52b683499ca47b5b#diff-2de3faa42cad092a3f1452f96bfbadca9b008ff43ebc477d8f0922a385c7d7bd). It was the cause why the missing method could not be called. When we use the constructor once on fetching level, we don't need it anymore in component levels

Reply:
Thank you so much! ❤️